packages/indexer-agent/src/indexer.ts

```diff
@@ -185,8 +178,12 @@ export class Indexer {
           if (result.error) {
             throw result.error
           }
-          this.logger.info('Proof of indexing returned', {
+          this.logger.debug('Reference Proof of indexing generated', {
```

Comment:
The capitalization here is weird. `Reference POI generated` or `Reference Proof Of Indexing generated` maybe?